### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by zzstoatzz**
while refining script in flow-metadata branch, adding views to enable block registration

### CODE REVIEW

**Review by ahuang11 — COMMENTED** [review]
Thanks! Looks good for first pass, I think we should add some pre-commit / black in this repo as well.

File: .github/workflows/merge-block-metadata.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
 
       - name: Merge collection metadata
```
[the remark this answers is not in the document]
Comment (zzstoatzz) [reply .github/workflows/merge-block-metadata.yml:21]: yes

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# Prefect Collection Registry
\ No newline at end of file
+# Prefect Collection Registry
+This is a registry of metadata for Prefect collections.
+
+Here we aggregate information about certain objects within collections, such as:
+- blocks defined in a collection
+- flows defined in a collection
```
[the remark this answers is not in the document]
Comment (zzstoatzz) [reply README.md:6]: i will, once its solidified, thats not the purpose of this PR

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    while repos_url:
+        print(f"Fetching {repos_url}")
+        response = httpx.get(repos_url, headers=headers)
+        response.raise_for_status()
+        repos = response.json()
+        for repo in repos:
+            if is_not_collection_repo(repo["name"]):
+                continue
+            yield repo["name"].replace("-", "_")
+        repos_url = response.links.get("next", {}).get("url")
```
[the remark this answers is not in the document]
Comment (zzstoatzz) [reply utils.py:51]: i like this idea, will incorporate

**Pull request closed without merging**